docs/sources/enterprise/activate-license.md

```diff
@@ -37,7 +45,20 @@ There are two different ways to add the license file to a Grafana instance:
 
   This setting can also be set with an environment variable, which is useful if you're running Grafana with Docker and have a custom volume where you have placed the license file. In this case, set the environment variable `GF_ENTERPRISE_LICENSE_PATH` to point to the location of your license file.
 
-## Step 3. Ensure that the license file's root url matches the root_url configuration option
+### Set the content of the license file on the *enterprise.license_text* configuration option
```

Comment:
Avoid italics in headings. You can just remove the file name from the heading. It is specified in the prose.

Reply:
```suggestion
### Set the content of the license file as a configuration option
```